torch/fx/passes/utils/matcher_utils.py

```diff
@@ -117,6 +125,17 @@ def _nodes_are_equal(self, pn: Node, gn: Node) -> bool:
         if not self.match_placeholder and pn.op == "placeholder":
             return True
 
+        if pn.op == "call_module" and self.match_flattened_modules:
+            # Check if gn is the last node of a flattened module that has the
+            # same type as the given pn call_module node's target
+
+            assert len(pn.args) == 1
+            module_type = type(self.pattern.owning_module.get_submodule(pn.target))
+
+            for module_nodes in self.module_partitions[module_type].values():
```

Comment:
so self.module_partitions[module_type] is a dictionary of `Dict[str, List[torch.fx.Node]]` so `.values` on that will just give you List[List[Node]]?